pkg/agent/api/delegatedidentity/v1/service.go

```diff
@@ -92,6 +96,10 @@ func (s *Service) isCallerAuthorized(ctx context.Context, log logrus.FieldLogger
 
 	for _, entry := range entries {
 		if _, ok := s.authorizedDelegates[entry.SpiffeId]; ok {
+			log.WithFields(logrus.Fields{
```

Comment:
above this line we may also want to add the selectors collected in another log, like
```
log.Error("no identity issued")
		return nil, status.Error(codes.PermissionDenied, "no identity issued")
```

Reply:
Added